compiler/extccomp.nim

```diff
@@ -1077,6 +1090,9 @@ proc writeJsonBuildInstructions*(conf: ConfigRef) =
 
     lit "\L}\L"
     close(f)
+    # we really should use json apis (eg %* and %) instead of all those `lit`,
+    # no need for premature optimization
+    writeFile $jsonFile, ($jsonFile).parseFile.pretty
```

Comment:
Remove this please, it has nothing to do with your patch and I don't like it.

Reply:
done